### PR TITLE
Add entity permissions tests

### PR DESCRIPTION
**Pull request by laceysanderson**
# Automated Testing
**This PR adds automated testing ONLY**

## Description
This PR adds automated testing for Tripal Content CRUD permissions. Specifically, `view [bundle name]`, `create [bundle name]`, `edit [bundle name]`, `delete [bundle name]`. I'm working on a Tripal Access Extension module and wanted to ensure I was not breaking core permissions. After developing the tests, I though I might as well contribute back :-) 

## Testing?
This will be automatically tested by Travis :-) 

There are two tests:
1) Simply tests that the permissions are available.
2) Tests that given a test entity and test user/role, users with the permission can access the content.

### CODE REVIEW

File: tests/tripal/entities/PermissionsTest.php [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+   * NOTE: We only test one bundle since it should be the same for all
+   * of them (done in a loop).
+   *
+   * @group permissions
+   */
+  public function testPermissionsForUser() {
+    $faker = Factory::create();
+
+    // Create organism entity for testing.
+    $bundle_name = 'bio_data_2';
```
Comment (bradfordcondon) [inline tests/tripal/entities/PermissionsTest.php]: I'm always scared assuming bundle number.  Can we instead look it up, o reven just double check the term_id make sense and if it doesnt exit with a warning?

**Review by bradfordcondon — APPROVED** [review]
Love to see new tests.  I read through these, great documentation.  I had one suggestion.  Thanks @laceysanderson

File: tests/tripal/entities/PermissionsTest.php (hunk 1 above)
Comment (laceysanderson) [reply tests/tripal/entities/PermissionsTest.php]: Good point! I looked it up now :-)

**Comment by laceysanderson** [comment]
Thank you for the review @bradfordcondon! Based on our current guidelines which say we only need "1 review by a contributor" I'm merging in these tests :-)

**Pull request merged**